spec/views/authors/show.html.erb_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "authors/show" do
+  context "when the author has a books" do
+    it "displays their books" do
+      author = create(:author, name: "Aaron")
```

Comment:
For a fast test suite it's good if the view specs don't hit the database. What do you think about about stubbing the association with something like:
```ruby
      author = build_stubbed(:author, name: "Aaron")
      book = build_stubbed(:book, author: author, title: "Legendary Ruby")
      allow(author).to receive(:books).and_return([book])

```